src/inmanta/data/__init__.py

```diff
@@ -1458,7 +1480,7 @@ def __init__(self, from_postgres=False, **kwargs):
     def _create_dict_wrapper(cls, from_postgres, kwargs):
         result = cls._create_dict(from_postgres, kwargs)
         new_messages = []
-        if from_postgres and "messages" in result:
+        if from_postgres and result.get("messages", None) is not None:
```

Comment:
None is implicit. This is actually the same. Additionally, the more pythonic way is then:

```suggestion
        if from_postgres and result.get("messages"):
```

Reply:
What do other think?